### ISSUE TITLE
KinesisFirehoseResponseRecord.Result value

### ISSUE THREAD

**Issue by elegos** [aws/aws-lambda-go#37]
Hello!
I'm tinkering with the new AWS lambda's go support. I created a very simple lambda function which (for ease of reading) simply passes-through the data.

According to the comment of [KinesisFirehoseResponseRecord.Result](https://github.com/aws/aws-lambda-go/blob/5fea2b254568d4043f23d27471ae265cd33d3530/events/firehose.go#L32)'s comment, the value should be one of `TransformedStateOk`, `TransformedStateDropped` or `TransformedStateProcessingFailed`.

https://github.com/aws/aws-lambda-go/blob/5fea2b254568d4043f23d27471ae265cd33d3530/events/firehose.go#L32

Problem is there is no such constant. The most similar ones are at [line 21-23](https://github.com/aws/aws-lambda-go/blob/5fea2b254568d4043f23d27471ae265cd33d3530/events/firehose.go#L21)

https://github.com/aws/aws-lambda-go/blob/5fea2b254568d4043f23d27471ae265cd33d3530/events/firehose.go#L21

Using these constants though Firehose complains about Result not being `Ok`, `Dropped` or `ProcessingFailed`.

Is it correct to assume that we *should* use the `KinesisFirehoseTransformedState*` constants and that they're bugged?

Thanks

```go
package main

import (
	"context"

	"github.com/aws/aws-lambda-go/events"
	"github.com/aws/aws-lambda-go/lambda"
)

func handler(ctx context.Context, event events.KinesisFirehoseEvent) (events.KinesisFirehoseResponse, error) {
	var response = events.KinesisFirehoseResponse{}

	for _, record := range event.Records {
		// For now a simple pass-through

		var transformedRecord = events.KinesisFirehoseResponseRecord{
			RecordID: record.RecordID,
			Result:   "Ok", // Here comes the problem
			Data:     record.Data,
		}

		response.Records = append(response.Records, transformedRecord)
	}

	return response, nil
}

func main() {
	lambda.Start(handler)
}

```

**Comment by cmello** [comment]
Thanks for opening this issue!

**Comment by bmoffatt** [comment]
resolved by #39
